runtime-doc/cups-pdf/autobuild/build

```diff
@@ -1,9 +1,24 @@
-cd src
-gcc ${CPPFLAGS} ${CFLAGS} ${LDFLAGS} -lcups -Wall -o cups-pdf cups-pdf.c
-install -D -m700 cups-pdf "$PKGDIR"/usr/lib/cups/backend/cups-pdf
+abinfo "Building cups-pdf ..."
+# FIXME: -Wl,-O1,--sort-common,--as-needed breaks build.
+#
+# /usr/bin/ld: /tmp/ccAmjmUt.o: in function `main':
+# .../cups-pdf-3.0.1/src/cups-pdf.c:834:(.text.startup+0x64): undefined
+# reference to `cupsBackendDeviceURI'
+# ...
+gcc \
+    ${CPPFLAGS} ${CFLAGS} ${LDFLAGS/-Wl,-O1,--sort-common,--as-needed/} \
+    -lcups -Wall \
+    -o "$SRCDIR"/cups-pdf \
+    "$SRCDIR"/src/cups-pdf.c
```

Comment:
No, you should move `-lcups` after `cups-pdf.c` to fix it.

Reply:
> No, you should move `-lcups` after `cups-pdf.c` to fix it.

Ah, thanks for the tip.

Reply:
Addressed in the latest revision.